test/unit/specs/fields/fieldNoUiSlider.spec.js

```diff
@@ -36,7 +36,71 @@ describe("fieldNoUiSlider.vue", () => {
 			expect(field.$el).to.be.exist;
 
 			expect(input).to.be.defined;
-			expect(input.classList.contains("slider")).to.be.true;			
+			expect(input.classList.contains("slider")).to.be.true;
+			expect(input.disabled).to.be.undefined;
+		});
+
+		before( () => {
+			vm.$appendTo(document.body);
+		});
+
+		it("should contain an handle element", (done) => {
+			if (window.noUiSlider) {
+				vm.$nextTick( () => {
+					let handle = input.querySelectorAll(".noUi-handle");
+					expect(handle.length).to.be.equal(1);					
+					expect(handle[0]).to.be.defined;
+					// expect(input.classList.contains("noui-target")).to.be.true;
+					done();
+				});
+			} else {
+				this.skip();
```

Comment:
No, the check is good, but change the `this.skip()` to `throw new Exception("Library is not loaded");`